result/isisiihs.yaml

```diff
@@ -0,0 +1,378 @@
+          x-field-uid: 2
+
+    IsisIIH.LocalGRLastAttemptStatus:
+      description: |-
+        This object contains the status of the last attempted Graceful Restart status of this router.
+        - succeeded: Choice is set if the last Graceful Status is successful.
+        - failed: The last Graceful Status is unsuccessful.
+        - inprogress: The last Graceful Restart status is in progress.
+        - unavailable: The last Graceful Restart status is not initiated.
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          x-enum:
+            succeeded:
+              x-field-uid: 1
+            failed:
+              x-field-uid: 2
+            inprogress:
+              x-field-uid: 3
+            unavailable:
+              x-field-uid: 4
+        succeeded:
+          $ref: '#/components/schemas/IsisIIH.LocalGRLastAttemptSucceeded'
+          x-field-uid: 2
+        failed:
+          $ref: '#/components/schemas/IsisIIH.LocalGRLastAttemptFailed'
+          x-field-uid: 3
+        inprogress:
+          type: boolean
+          x-field-uid: 4
+        unavailable:
+          type: boolean
+          x-field-uid: 5
+
+    IsisIIH.LocalGRLastAttemptFailed:
+      description: >-
+        This container contains the failure status of the last Graceful Restart initiated by this router.
+      type: object
+      properties:
+        reason:
+          description: |-
+            Failure reason of last Graceful Restart.
+          type: string
+          x-field-uid: 1
+
+    IsisIIH.LocalGRLastAttemptSucceeded:
+      description: >-
+        This container contains details about the successful status of the last Graceful Restart initiated by this router.
+      type: object
+      properties:
+        lsdb_syncup_time:
+          description: |-
+            The time is taken to synchronize the L1 and L2 LSDB by this Restarting router.
+            By this time, the CSNP list is cleared and all LSPs are collected by the neighbor(s).
+          type: integer
+          format: uint32
+          x-field-uid: 1
+        adjacency_bring_up_time:
+          description: |-
+            The time (in second) is taken to bring up the adjacency by this Restarting router.
+            The time is counted when the Restart TLV with RR bit set in a IIH PDU is sent 
+            and till start sending the Restart TLV with RR bit unset.
+          type: integer
+          format: uint32
+          x-field-uid: 3
+
+    IsisIIH.NeighborGRLastAttemptStatus:
+      description: >-
+        This object contains the status of the last successful Graceful Restart status of a neighbor router.
+      type: object
+      properties:
+        adjacency_bring_up_time:
+          description: |-
+            The time (in second) is taken to bring up adjacency by the Restarting Neighbor.
+            The time is counted by a Helper router when the Restart TLV with RR bit set in a IIH PDU is received 
```

Comment:
The time is measured from when a Helper router initially receives a Restart TLV with RR bit set in an IIH PDU upto the time when it receives the Restart TLV with RR bit unset in an IIH PDU from the Neighbor Router.